cockatrice/src/deck/deck_loader.cpp

```diff
@@ -481,6 +481,52 @@ void DeckLoader::saveToStream_DeckZoneCards(QTextStream &out,
     }
 }
 
+bool DeckLoader::convertToCockatriceFormat(QString fileName)
+{
+    // Change the file extension to .cod
+    QFileInfo fileInfo(fileName);
+    QString newFileName = fileInfo.path() + "/" + fileInfo.completeBaseName() + ".cod";
```

Comment:
This might not be true on all OS'es, might want to use the fileInfo or QDir::toNativeSeparators